docs/image-based-upgrade.md

```diff
@@ -291,9 +291,11 @@ oc patch imagebasedupgrade upgrade -n openshift-lifecycle-agent --type='json' -p
 The "Prep" stage will:
 
 - Perform the following validations:
-  - If the oadpContent is populated, validate that the specified configmap has been applied and is valid
-  - If the extraManifests is populated, validate that the specified configmap has been applied and is valid
-    - If a required CRD is missing from the current stateroot, a warning message will be included in the IBU CRs with annotation `lca.openshift.io/warn-extramanifest-cm-unknown-crd`.
+  - If the `oadpContent` is populated, validate that the specified configmap has been applied and is valid
+  - If the `extraManifests` is populated, validate that the specified configmap has been applied and is valid
+    - Validation errors from Dry-run apply are treated as warnings. This also includes cases where CRDs are missing from the current stateroot and dependent namespace does not exist on the current stateroot but is also not found in the configmaps.
+    LCA doesn't block the upgrade due to validation warnings. Instead, it updates the Prep status condition with a warning message and annotates IBU with the annotation `lca.openshift.io.extramanifest-cm-validation-warning`, providing detailed information about the failures
```

Comment:
What do you think about updating annotation to something like
```
extra-manifest.lca.openshift.io/warn-validation-error
```

Follows the recommended pattern `subsystem.lca.ocp.io/name`  and matches our other ones we have (autorollback and upcoming HC one too)